sdk/textanalytics/Azure.AI.TextAnalytics/src/AnalyzeHealthcareEntitiesOperation.cs

```diff
@@ -288,7 +226,7 @@ public virtual void Cancel(CancellationToken cancellationToken = default)
             try
             {
                 ResponseWithHeaders<TextAnalyticsCancelHealthJobHeaders> response = _serviceClient.CancelHealthJob(new Guid(Id), cancellationToken);
-                _response = response.GetRawResponse();
+                _operationInternal.RawResponse = response.GetRawResponse();
```

Comment:
Wonder is this method should've returnd a Response.

Reply:
This change hasn't GAed yet, so it can be discussed.

Reply:
what will be the advantage of returning a Response? Should we not use the `Operation.Response` property for it?

Reply:
Not sure what the original reasoning was, but that's how `UpdateStatus` behaves as well: it sets the operation's raw response, but returns it as `Response` from the method call as well.

Not really related to this PR, and no strong opinions from my side.